src/Plivo/Resource/MultiPartyCall/MultiPartyCallInterface.cs

```diff
@@ -686,7 +735,141 @@ public DeleteResponse<MultiPartyCall> Stop(string mpcUuid = null, string friendl
+            string fileFormat = "mp3", 
+            string statusCallbackUrl = null, string statusCallbackMethod = "POST"
+            ) 
+        {
+            if (mpcUuid != null)
+            {
+                MpcUtils.ValidParamString("mpcUuid", mpcUuid, false);
+            }
+            if (friendlyName != null)
+            {
+                MpcUtils.ValidParamString("friendlyName", friendlyName, false);
+            }
+
+            if (fileFormat != null)
+            {
+                MpcUtils.ValidParamString("fileFormat", fileFormat.ToLower(), false,
+                    new List<string>() {"mp3", "wav"});
+            }
+
+            if (statusCallbackUrl != null)
+            {
+                MpcUtils.ValidUrl("statusCallbackUrl", statusCallbackUrl, false);
+            }
+
+            if (statusCallbackMethod != null)
+            {
+                MpcUtils.ValidParamString("statusCallbackMethod", statusCallbackMethod.ToUpper(), false,
+                    new List<string>() {"GET", "POST"});
+            }
+            MpcUtils.ValidParamString("participantId", participantId, true);
+            string mpcId = MakeMpcId(mpcUuid, friendlyName);
+            var mandatoryParams = new List<string> { "" };
+            bool isVoiceRequest = true;
+            var data = CreateData (
+                mandatoryParams,
+                new {
+                    fileFormat,
+                    statusCallbackUrl,
+                    statusCallbackMethod,
+                    isVoiceRequest
+                });
+
+            return ExecuteWithExceptionUnwrap (() => {
+                return Task.Run(async () =>
+                    await UpdateSecondaryResource<RecordCreateResponse<MultiPartyCallParticipant>>(mpcId,
+                        data,
+                        "Participant", participantId + "/Record").ConfigureAwait(false)).Result;
+            });
+        }
+        
+        public DeleteResponse<MultiPartyCallParticipant> StopParticipantRecording (string participantId, string mpcUuid = null, string friendlyName = null) {
+            if (mpcUuid != null)
+            {
+                MpcUtils.ValidParamString("mpcUuid", mpcUuid, false);
+            }
+            if (friendlyName != null)
+            {
+                MpcUtils.ValidParamString("friendlyName", friendlyName, false);
+            }
+            MpcUtils.ValidParamString("participantId", participantId, true);
+            string mpcId = MakeMpcId(mpcUuid, friendlyName);
+            var mandatoryParams = new List<string> { "" };
+            bool isVoiceRequest = true;
+            var data = CreateData (
+                mandatoryParams, new { isVoiceRequest });
+
+            return ExecuteWithExceptionUnwrap (() => {
+                return Task.Run(async () => await DeleteSecondaryResource<DeleteResponse<MultiPartyCallParticipant>>(mpcId,
+                    new Dictionary<string, object> () { {"is_voice_request", true} }, 
```

Comment:
`is_voice_request` is not required since we have already given `isVoiceRequest` in line 806